zappa/core.py

```diff
@@ -2098,12 +2098,17 @@ def get_rest_apis(self, project_name):
         """
         Generator that allows to iterate per every available apis.
         """
-        all_apis = self.apigateway_client.get_rest_apis(limit=500)
+        pagination_position = "init"
```

Comment:
No big issues, however, in terms of code clarity I'm a little adverse to re-purposing an variable.

There are two conditions being evaluated here, if it's the initial and then if there are additional pages.
It's my preference that these two conditions are managed in separate variables, perhaps something like:
```
next_position = None
is_initial = False
while next_position or is_initial:
    response = self.apigateway_client.get_rest_apis(limit=500, position=next_position)
    yield from (api for api in response.get("items", []) if api["name"] == project_name)

    is_initial = False
    next_position = response.get("position", None)
   
    
```